### ISSUE TITLE
training loss is 0 in PAI but not in pure docker on PC

### ISSUE THREAD

**Issue by sophia-hxw** [microsoft/pai#5186]
**Organization Name**:

**Short summary about the issue/question**:
loss = 0 when I was training on PAI, but loss looks like normal when I used the same docker with PAI on my PC

[training code](https://github.com/meteorshowers/RCF-pytorch)

**Brief what process you are following**:

docker: pytorch:py3-pytorch_1.6.0-10.2-cudnn7

PAI training log example:【stage loss and img loss was added by me in the training process】
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
img loss:  tensor([0.], grad_fn=<CopyBackwards>)
stage loss:  tensor([0.])
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
stage loss:  tensor([0.], grad_fn=<CopyBackwards>)
img loss:  tensor([0.], grad_fn=<CopyBackwards>)

training log on PC:
stage loss:  tensor([2031.5410], grad_fn=<CopyBackwards>)
stage loss:  tensor([4965.3057], grad_fn=<CopyBackwards>)
stage loss:  tensor([11768.6533], grad_fn=<CopyBackwards>)
stage loss:  tensor([18030.7305], grad_fn=<CopyBackwards>)
stage loss:  tensor([20348.7227], grad_fn=<CopyBackwards>)
img loss:  tensor([23027.8340], grad_fn=<CopyBackwards>)
Epoch: [0/30][0/6] Time 0.771 (avg:0.771) Loss 2302.783447 (avg:2302.783447)
stage loss:  tensor([0.])
stage loss:  tensor([2294.5977], grad_fn=<CopyBackwards>)
stage loss:  tensor([4602.5137], grad_fn=<CopyBackwards>)
stage loss:  tensor([13410.5117], grad_fn=<CopyBackwards>)
stage loss:  tensor([17896.5195], grad_fn=<CopyBackwards>)
stage loss:  tensor([19977.5020], grad_fn=<CopyBackwards>)
img loss:  tensor([22320.6523], grad_fn=<CopyBackwards>)

**How to reproduce it**:

**OpenPAI Environment**:
- OpenPAI version: 1.3.0
- Cloud provider or hardware configuration:
- OS (e.g. from /etc/os-release):
- Kernel (e.g. `uname -a`):
- Hardware (e.g. core number, memory size, storage size, GPU type etc.):
- Others:

**Anything else we need to know**:

**Comment by sophia-hxw** [comment]
resolved
